src/stactools/met_office_deterministic/constants.py

```diff
@@ -0,0 +1,671 @@
+        in physics. Upwards is positive; negative is downward.""",
+        "unit": "W m-2",
+    },
+    "snowfall_accumulation": {
+        "description": """Implied depth of snow (as liquid water equivalent) which has
+        been deposited on the surface.""",
+        "unit": "m",
+    },
+}
+
+# Global Deterministic Variables by Collection Type
+
+global_pressure_variables = {
+    **_shared_pressure_variables,
+    **_global_only_pressure_variables,
+}
+
+global_height_variables = _shared_height_variables.copy()
+
+global_surface_variables = {
+    **_shared_surface_variables,
+    **_global_only_surface_variables,
+}
+
+# UK Deterministic Variables by Collection Type
+
+uk_pressure_variables = _shared_pressure_variables.copy()
+
+uk_height_variables = {
+    **_shared_height_variables,
+    **_uk_only_height_variables,
+}
+
+uk_surface_variables = {
+    **_shared_surface_variables,
+    **_uk_only_surface_variables,
+}
+
+GLOBAL_BBOX = [-180.0, -90.0, 180.0, 90.0]
+GLOBAL_GEOMETRY = {
+    "type": "Polygon",
+    "coordinates": [
+        [
+            [-180.0, -90.0],
+            [180.0, -90.0],
+            [180.0, 90.0],
+            [-180.0, 90.0],
+            [-180.0, -90.0],
+        ]
+    ],
+}
+
+UK_PROJECTED_BBOX = [-1159000.0, -1037000.0, 925000.0, 903000.0]
+
+UK_PROJECTED_GEOMETRY = {
+    "type": "Polygon",
+    "coordinates": [
+        [
+            [-1159000.0, -1037000.0],
+            [925000.0, -1037000.0],
+            [925000.0, 903000.0],
+            [-1159000.0, 903000.0],
+            [-1159000.0, -1037000.0],
+        ]
+    ],
+}
+UK_PROJECTED_CRS_WKT2 = """PROJCRS["unnamed",
+    BASEGEOGCRS["unknown",
+        DATUM["unnamed",
+            ELLIPSOID["Spheroid",6378137,298.257222101004,
+                LENGTHUNIT["metre",1,
+                    ID["EPSG",9001]]]],
+        PRIMEM["Greenwich",0,
+            ANGLEUNIT["degree",0.0174532925199433,
+                ID["EPSG",9122]]]],
+    CONVERSION["unnamed",
+        METHOD["Lambert Azimuthal Equal Area",
+            ID["EPSG",9820]],
+        PARAMETER["Latitude of natural origin",54.9,
+            ANGLEUNIT["degree",0.0174532925199433],
+            ID["EPSG",8801]],
+        PARAMETER["Longitude of natural origin",-2.5,
+            ANGLEUNIT["degree",0.0174532925199433],
+            ID["EPSG",8802]],
+        PARAMETER["False easting",0,
+            LENGTHUNIT["metre",1],
+            ID["EPSG",8806]],
+        PARAMETER["False northing",0,
+            LENGTHUNIT["metre",1],
+            ID["EPSG",8807]]],
+    CS[Cartesian,2],
+        AXIS["easting",east,
+            ORDER[1],
+            LENGTHUNIT["metre",1,
+                ID["EPSG",9001]]],
+        AXIS["northing",north,
+            ORDER[2],
+            LENGTHUNIT["metre",1,
+                ID["EPSG",9001]]]]
+"""
+UK_BBOX = [-24.53378493833058, 44.50650694725796, 15.303254906837337, 63.01353038140843]
```

Comment:
Nit: this is a pretty high level of [precision](https://xkcd.com/2170/). Unless these values are taken from a canonical source (e.g. the UK's own definitions), I'd dial it back to 7-9 digits?